web/models/package_download.ex

```diff
@@ -2,11 +2,11 @@ defmodule HexWeb.PackageDownload do
   use HexWeb.Web, :model
   import HexWeb.QueryAPI, only: [coalesce: 2]
 
-  @primary_key false
```

Comment:
We need primary keys here in order to identify uniqueness when handling associations. If this primary key is not good enough, we will need to add one to the schema.